docs/running-workflows/supported-systems/snakemake/index.md

```diff
@@ -0,0 +1,3 @@
+# Snakemake
+
+A workflow management system originated in bioinformatics.
```

Comment:
Maybe we can enrich this part a little bit? Although it can be part of another issue together with the rest of the workflow languages.

Reply:
Yeah, I left it like this because the others were very short as well. Do you reckon I should elaborate on Snakemake in this PR then we can create an issue for the other engines?

Reply:
That makes sense! Please go ahead, thanks!